examples/CORE/mm.yaml

```diff
@@ -0,0 +1,49 @@
+programs:
+  - name: kswapd
+    metrics:
+      counters:
+        - name: kswapd
+          help: kswapd wake count
+          table: kswapd_counts
+          labels:
+            - name: numa_node
+              size: 4
+              decoders:
+                - name: uint
+            - name: numa_zone
+              size: 4
+              decoders:
+                - name: uint
+                - name: static_map
+                  static_map:
+                    0: DMA
+                    1: DMA32
+                    2: NORMAL
+                    3: MOVEABLE
+                    4: DEVICE
+    tracepoints:
+      tracepoint__vmscan__mm_vmscan_kswapd_wake: "vmscan:mm_vmscan_kswapd_wake"
+  - name: kcompactd
+    metrics:
+      counters:
+        - name: kcompactd_wake_count
+          help: kcompactd wake count
+          table: kcompactd_counts
+          labels:
+            - name: numa_node
+              size: 4
+              decoders:
+                - name: uint
+            - name: numa_zone
+              size: 4
+              decoders:
+                - name: uint
+                - name: static_map
+                  static_map:
+                    0: DMA
+                    1: DMA32
+                    2: NORMAL
+                    3: MOVEABLE
+                    4: DEVICE
+    tracepoints:
+      "compaction:mm_compaction_kcompactd_wake" tracepoint__compaction__mm_compaction_kcompactd_wake
```

Comment:
Looks like there's a missing `:` here. If you don't need the quotes around the keys, I'd recommend to drop them.

Reply:
1. Add the `:` 
2.  Remove the quotes